Scripts/SDL/Garrison.sdl

```diff
@@ -912,3 +912,164 @@ STATEDESC Garrison
+    VAR BOOL    grsnJourneyCloth05Vis[1]    DEFAULT=1 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnJourneyCloth06Vis[1]    DEFAULT=1 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnJourneyCloth07Vis[1]    DEFAULT=1 DEFAULTOPTION=VAULT
+
+    VAR BOOL    grsnTreasureBook02Vis[1]    DEFAULT=1 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnTreasureBook11Vis[1]    DEFAULT=1 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnYeeshaPage05Vis[1]      DEFAULT=1 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnYeeshaPage02Vis[1]      DEFAULT=1 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnTrnCtrRotationOn[1]     DEFAULT=1 DEFAULTOPTION=VAULT
+
+    VAR BOOL    grsnNexusBookVis[1]         DEFAULT=1 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnWallInfoJournalVis[1]   DEFAULT=0 DEFAULTOPTION=VAULT
+
+    VAR BOOL    grsnBahroStoneToShaftVis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+
+    VAR BOOL    grsnWellFirstFloorBlocker[1] DEFAULT=0 DEFAULTOPTION=VAULT
+
+    VAR BOOL    grsnCalendarSpark01[1] DEFAULT=0 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnCalendarSpark04[1] DEFAULT=0 DEFAULTOPTION=VAULT
+    
+    VAR BOOL    grsnGrantMaintainerSuit[1] DEFAULT=0 DEFAULTOPTION=VAULT
+
+
+
+# Age Mechanics
+    VAR BOOL    grsnWellMainPowerOn[1]    DEFAULT=0 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnBahroDoorClosed[1]    DEFAULT=1
+
+    VAR BOOL    grsnMainSwitchOn[1]     DEFAULT=0 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnGearSwitchOn[1]     DEFAULT=0 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnUpElevSwitchOn[1]       DEFAULT=0 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnDnElevSwitchOn[1]       DEFAULT=0 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnDoorSwitchOn[1]     DEFAULT=0 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnGearBrake01On[1]        DEFAULT=1 DEFAULTOPTION=VAULT
+    VAR BOOL    grsnGearBrake02On[1]        DEFAULT=1 DEFAULTOPTION=VAULT
+
+# prison variables
+
+    VAR BOOL grsnPrisonBones01vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonBones02vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonBowls01vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonBowls02vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonBowls03vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonBowls04vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonBowls05vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonChains01vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonChains02vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonDirt01vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonDirt02vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonMattress01vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonMattress02vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonWindow01vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+        VAR BOOL grsnPrisonWindow02vis[1] DEFAULT=0 DEFAULTOPTION=VAULT
+    VAR BOOL grsnYeeshaPage02VisDUP[1] DEFAULT=0 DEFAULTOPTION=VAULT
```

Comment:
If we're making a new version, we should simply delete this variable IMO.

Reply:
Per the v28 originator, there is concern/uncertainty that MOULa's server code will correctly handle deletion of a variable from the SDL between versions, thus the duplicate was only renamed instead.